fusor-ember-cli/app/routes/deployment.js

```diff
@@ -13,6 +13,8 @@ export default Ember.Route.extend(DeploymentRouteMixin, {
     controller.set('organizationTabRouteName', 'configure-organization');
     controller.set('lifecycleEnvironmentTabRouteName', 'configure-environment');
     controller.set('model.host_naming_scheme', 'Freeform');
+    controller.set('confirmRhevRootPassword', model.get('rhev_root_password'));
+    controller.set('confirmRhevEngineAdminPassword', model.get('rhev_engine_admin_password'));
```

Comment:
@cfchase, this is the major fix as `confirmRhevRootPassword` and `confirmRhevRootPassword` are set on the top deployment route as opposed to deployment/rhev/rhev-options, but `validRhev` needs these confirm to verify it is valid. And `isDisabledSubscriptions` is dependant on `validRhev`